.gitignore

```diff
@@ -16,6 +16,8 @@ tfproviderdocsgen
 # OS generated files
 .DS_Store
 
+vendor
```

Comment:
Why did you feel the need to add this here, given we build with Go 1.17?

Reply:
I still use go mod vendor, which is probably bad practice.  I can remove.